src/components/Planetarium/Planetarium.tsx

```diff
@@ -3,15 +3,34 @@ import { PlanetBio, AllData } from '../../interface';
 import './planetarium.css';
 import Planet from '../Planet/Planet';
 
-const Planetarium: React.FC<AllData> = ({ allPlanets }): JSX.Element => {
+const Planetarium: React.FC<AllData> = ({ allPlanets, sortKey }): JSX.Element => {
+
+  const addDescriptor = (data: any, key: any) => {
+    let descriptor: string = '';
+    if (key === 'distance_from_sun' || key === 'diameter') {
+      descriptor = 'kilometers';
+    } else if ( key === 'mass') {
+      descriptor = 'kilograms';
+    } else if ( key === 'length_of_day') {
+      descriptor = 'hours';
+    } else if ( key === 'length_of_year') {
+      descriptor = 'days'
+    } else if ( key === 'number_of_moons') {
+      descriptor = 'moons'
+    } else if ( key === 'gravity') {
+      descriptor = 'metres per second squared'
```

Comment:
Minorest of minor nitpicks here but let's make it "kilometers" and "meters" or "kilometres" and "metres"